bsp/m5_atom_s3/m5_atom_s3.c

```diff
@@ -0,0 +1,344 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "driver/gpio.h"
+#include "driver/ledc.h"
+#include "driver/spi_master.h"
+#include "esp_err.h"
+#include "esp_log.h"
+#include "esp_check.h"
+#include "esp_spiffs.h"
+#include "esp_lcd_panel_io.h"
+#include "esp_lcd_panel_vendor.h"
+#include "esp_lcd_panel_ops.h"
+#include "esp_vfs_fat.h"
+
+#include "button_gpio.h"
+#include "bsp/m5_atom_s3.h"
+#include "bsp/display.h"
+#include "esp_lcd_gc9a01.h"
+#include "esp_lvgl_port.h"
```

Comment:
```suggestion
```

Here we unconditionally include esp_lvgl_port. But it should depend on NOGLIB settings.

Easiest way is to remove this line and take the include from `bsp/m5_atom_s3.h`

Reply:
Fixed.